src/main/java/org/wso2/securevault/secret/repository/FileBaseSecretRepository.java

```diff
@@ -84,8 +84,8 @@ public void init(Properties properties, String id) {
 
         Properties cipherProperties = MiscellaneousUtil.loadProperties(filePath);
         if (cipherProperties.isEmpty()) {
-            if (log.isDebugEnabled()) {
-                log.debug("Cipher texts cannot be loaded form : " + filePath);
+            if (log.isInfoEnabled()){
```

Comment:
why do we checking log.info enabled

Reply:
Revert Back to the old implementation -> Debug Logs